documentation/dev-portal/src/events/cryptotalk/nym-vpn-faq.md

```diff
@@ -1,6 +1,6 @@
-# Frequently Asked Questions @ 37C3
+# Frequently Asked Questions
 
-This is a FAQ page tailored for CCC 2023 event (37c3). If you interested to read more about Nym platform, you can have a look at [Nym general FAQ](https://nymtech.net/developers/faq/general-faq.html) and read through Nym's technical [documentation](https://nymtech.net/docs), [Developer Portal](https://nymtech.net/developers) and [Operators Guide](https://nymtech.net/operators).
+This is a FAQ page tailored for this event. If you interested to read more about Nym platform, you can have a look at [Nym general FAQ](https://nymtech.net/developers/faq/general-faq.html) and read through Nym's technical [documentation](https://nymtech.net/docs), [Developer Portal](https://nymtech.net/developers) and [Operators Guide](https://nymtech.net/operators).
```

Comment:
have to attach this to this line, because of the github UI but: 
* ln9 you have a typo: just 'FAQs', no apostrophe
* ln13: "through" and "layers" have typos